src/test/java/org/broadinstitute/hellbender/tools/spark/sv/ContigAlignerTest.java

```diff
@@ -0,0 +1,137 @@
+        contigsData.add(">contig-14 151 0");
+        contigsData.add("TAGAATTTTGGATGGGTGCAAGTCAAGTAATGGGATAAAAGAATGATGATCGAAGGAGATGATTCATGATAACTATGTGCCCTTATGTGAAACCTAGCAACAATTTCTTACTTATGATAAAGGCACAATCTCCCCAGTCCAGAATCTAAAT");
+        contigsData.add(">contig-7 151 0");
+        contigsData.add("TATTAAATAAAAGTGGTAGAATTTTGGATGGGTGCAAGTCAAGTAATGGGATAAAAGAATGATGATCGAAGGAGATGACTCATGATAACTAAGTGCCCTTATGTGAAACCTAGCAACAATTTCTTACTTATGATAAAGGCACAATCTCCCC");
+
+        final RunSGAViaProcessBuilderOnSpark.ContigsCollection contigsCollection = new RunSGAViaProcessBuilderOnSpark.ContigsCollection(contigsData);
+
+        final ContigAligner contigAligner = new ContigAligner(new File(b37_reference_20_21));
+        final List<ContigAligner.AssembledBreakpoint> assembledBreakpoints = contigAligner.alignContigs(contigsCollection);
+        Assert.assertEquals(2, assembledBreakpoints.size());
+
+        final ContigAligner.AssembledBreakpoint breakpoint1 = assembledBreakpoints.get(0);
+
+        Assert.assertEquals(breakpoint1.contigId, ">contig-9 2009 0");
+
+        final ContigAligner.AlignmentRegion breakpoint1Region1 = breakpoint1.region1;
+        //todo verify that the 1-based vs 0-based nature of this is correct
+        Assert.assertEquals(breakpoint1Region1.interval, new SimpleInterval("21", 27373209, 27374159));
+        Assert.assertTrue(breakpoint1Region1.forwardStrand);
+        Assert.assertEquals(breakpoint1Region1.mqual, 60);
+
+        final ContigAligner.AlignmentRegion breakpoint1Region2 = breakpoint1.region2;
+        Assert.assertEquals(breakpoint1Region2.interval, new SimpleInterval("21", 27374159, 27374707));
+        Assert.assertFalse(breakpoint1Region2.forwardStrand);
+        Assert.assertEquals(breakpoint1Region2.mqual, 60);
+
+        Assert.assertEquals(breakpoint1.homology, "GGATCCA");
+        Assert.assertEquals(breakpoint1.insertedSequence, "NA");
+
+        final ContigAligner.AssembledBreakpoint breakpoint2 = assembledBreakpoints.get(1);
+
+        Assert.assertEquals(breakpoint2.contigId, ">contig-9 2009 0");
+
+        final ContigAligner.AlignmentRegion breakpoint2Region1 = breakpoint2.region1;
+        Assert.assertEquals(breakpoint2Region1.interval, new SimpleInterval("21", 27374159, 27374707));
+        Assert.assertFalse(breakpoint2Region1.forwardStrand);
+        Assert.assertEquals(breakpoint2Region1.mqual, 60);
+
+        final ContigAligner.AlignmentRegion breakpoint2Region2 = breakpoint2.region2;
+        Assert.assertEquals(breakpoint2Region2.interval, new SimpleInterval("21", 27374701, 27375219));
+        Assert.assertTrue(breakpoint2Region2.forwardStrand);
+        Assert.assertEquals(breakpoint2Region2.mqual, 60);
+
+        Assert.assertEquals(breakpoint2.homology, "NA");
+        Assert.assertEquals(breakpoint2.insertedSequence, "NA");
+
+
```

Comment:
I think `contigAligner.close()` is on call here. Same thing goes for the other test. Or have a `@AfterClass`.